changelog/1897.removal.rst

```diff
@@ -0,0 +1 @@
+Removed default value of ``None`` for the ``request_context`` parameter of ``urllib3.poolmanager.PoolManager.connection_from_pool_key``.
```

Comment:
Since `PoolManager` is exposed at `urllib3.PoolManager` let's use that as our reference

Reply:
Done!